cookbook/openfold3_to_openfe.ipynb

```diff
@@ -0,0 +1,1053 @@
+    "  - Fixing & protonating the reference protein structure.\n",
+    "\n",
+    "3. Loading structures into OpenFE Components\n",
+    "\n",
+    "4. Creating an Absolute Binding Free Energy Transformation\n",
+    "\n",
+    "5. Creating a SepTop Relative Binding Free Energy Transformation\n",
+    "\n",
+    "6. Creating a Hybrid Topology Relative Binding Free Energy Transformation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2fea29c3",
+   "metadata": {},
+   "source": [
+    "## 1. Run cofolding with OpenFold3 on a set of TYK2 ligands\n",
+    "\n",
+    "### Preparing the Input JSON for OpenFold\n",
+    "\n",
+    "First we need to create an input JSON file for OpenFold. This step requires assembling two key pieces of information:\n",
+    "\n",
+    "- Ligand SMILES strings\n",
+    "- Protein sequence (extracted from the PDB file)\n",
+    "\n",
+    "In this workflow, we begin with an SDF file containing the ligands.\n",
+    "We extract the SMILES representation for each ligand and store them in a dictionary alongside their ligand identifiers."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "08962d91-a334-4c89-adf1-19be6f2bdb16",
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "ligands_file = \"assets/tyk2_ligands.sdf\"\n",
+    "from rdkit.Chem.Descriptors3D import Asphericity\n",
+    "from rdkit import Chem\n",
+    "\n",
+    "ligands_dict = {}\n",
+    "sppl = Chem.SDMolSupplier(ligands_file, removeHs=True)\n",
+    "for mol in sppl:\n",
+    "    smi = Chem.MolToSmiles(mol)\n",
+    "    name = mol.GetProp(\"_Name\")\n",
+    "    ligands_dict[name] = smi"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "1c55bfeb-c1bc-4af8-98da-43ff8c1de141",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'lig_ejm_31': 'CC(=O)Nc1cc(NC(=O)c2c(Cl)cccc2Cl)ccn1',\n",
+       " 'lig_ejm_42': 'CCC(=O)Nc1cc(NC(=O)c2c(Cl)cccc2Cl)ccn1',\n",
+       " 'lig_ejm_43': 'CC(C)C(=O)Nc1cc(NC(=O)c2c(Cl)cccc2Cl)ccn1',\n",
+       " 'lig_ejm_46': 'O=C(Nc1ccnc(NC(=O)C2CC2)c1)c1c(Cl)cccc1Cl',\n",
+       " 'lig_ejm_47': 'O=C(Nc1ccnc(NC(=O)C2CCC2)c1)c1c(Cl)cccc1Cl',\n",
+       " 'lig_ejm_48': 'O=C(Nc1ccnc(NC(=O)C2CCCC2)c1)c1c(Cl)cccc1Cl',\n",
+       " 'lig_ejm_50': 'O=C(CO)Nc1cc(NC(=O)c2c(Cl)cccc2Cl)ccn1',\n",
+       " 'lig_jmc_23': 'O=C(Nc1ccnc(NC(=O)[C@H]2C[C@H]2F)c1)c1c(Cl)cccc1Cl',\n",
+       " 'lig_jmc_27': 'O=C(Nc1ccnc(NC(=O)[C@H]2C[C@H]2Cl)c1)c1c(Cl)cccc1Cl',\n",
+       " 'lig_jmc_28': 'C[C@@H]1C[C@@H]1C(=O)Nc1cc(NC(=O)c2c(Cl)cccc2Cl)ccn1'}"
+      ]
+     },
+     "execution_count": 2,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "ligands_dict"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "80da7967-8046-44e4-aef7-c81ac6712f98",
+   "metadata": {},
+   "source": [
+    "We then create the json file that will be the OpenFold input."
```

Comment:
```suggestion
    "We then create the JSON file that will be the OpenFold input."
```

Reply:
It might be nice to link to relevant OpenFold docs, but maybe not if they aren't stable yet. Broken link is worse than no link.